src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -82,7 +83,7 @@ internal sealed class MemberSignatureComparer : IEqualityComparer<Symbol>
             considerReturnType: false,
             considerTypeConstraints: false,
             considerCallingConvention: false,
-            considerRefKindDifferences: true,
+            refKindCompareMode: RefKindCompareMode.ConsiderDifferences | RefKindCompareMode.AllowRefReadonlyVsInMismatch,
```

Comment:
Do we have a test showing the impact of this change?

Reply:
Adding `Implementation_RefReadonly_In_Close`.

Reply:
Thanks for the follow-up